mlflow/models/evaluation/default_evaluator.py

```diff
@@ -264,11 +266,11 @@ def gen_line_x_y_label_fn(_y, _y_prob):
 
         def gen_line_x_y_label_fn(_y, _y_prob):
             precision, recall, _thresholds = sk_metrics.precision_recall_curve(_y, _y_prob)
-            ap = np.mean(precision)
+            ap = sk_metrics.average_precision_score(_y, _y_prob)
```

Comment:
Do we need to pass `pos_label` to `average_precision_score`?

Reply:
Yep!